include/f_core/net/application/c_udp_broadcast_tenant.h

```diff
@@ -45,9 +47,27 @@ class CUdpBroadcastTenant {
     void TransmitMessageAsynchronous() {
         T message{};
         if (messagesToBroadcast->Receive(message, K_NO_WAIT) == 0) {
-            udp.TransmitAsynchronous(message, sizeof(T));
+            udp.TransmitAsynchronous(&message, sizeof(T));
         }
     }
+
+    /**
+     * See parent docs
+     */
+    void Startup() override {};
```

Comment:
It looks like Startup and PostStartup have empty defaults - do we gain anything from overriding them to do the same thing (nothing)?

Reply:
Have to for a compile, since parent class functions are virtual.